Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build stage
-FROM rust:1.72-alpine3.18 AS cargo-build
+FROM docker.io/rust:1.72-alpine3.18 AS cargo-build
```

Comment:
Without this change, podman on ubuntu arm64 returns the below error:

> Error: creating build container: short-name "rust:1.72-alpine3.18" did not resolve to an alias and no unqualified-search registries are defined in "/etc/containers/registries.conf"

Reply:
Is it some transient error that will go away on itself?

Is it the same for other Rust or Alpine versions?

Reply:
Does short name always mean "docker.io" or it can mean some "default container registry" e.g. "ghcr.io" when on Github?

Reply:
It also depends on the registry list provided in the `/etc/containers/registries.conf`. The first registry where the image is found will be used.

Reply:
So maybe specifying `docker.io` explicitly can be restrictive for those who want to avoid that registry?

Is Alpine+Rust represented only on docker.io or in other registries as well?

Reply:
I have amended the podman install step to add 'docker.io' into the docker image search registry. The image name change is dropped now.

Reply:
ping for another review :)